### ISSUE TITLE
Missing attributes for HiQ SimulatorMPI

### ISSUE THREAD

**Issue by Neuromancer43** [Huawei-HiQ/HiQsimulator#17]
I noticed that some features of backend `SimulatorMPI` given in the [HiQ documentation](https://hiqsimulator.readthedocs.io/en/latest/hiq.projectq.backends.html) don't work.

For example:
`engine.backend.get_expectation_value()`  is missing, with error message:

cpuset: 0x00000003 nodeset: 0xf...f Traceback (most recent call last): File "***/nfs/hiq/mpi/SOUT1564222561662", line 52, in <module> expectation=eng.backend.get_expectation_value(operator,qb_array) File "***/.local/lib/python3.5/site-packages/HiQsimulator-0.0.1-py3.5-linux-x86_64.egg/hiq/projectq/backends/_sim return self._simulator.get_expectation_value(operator, AttributeError: 'hiq.projectq.backends._sim._cppsim_mpi.SimulatorMP' object has no attribute 'get_expectation_value' 

while the same test code works correctly with ProjectQ backend simulator.

**Comment by Takishima** [comment]
The function is indeed missing. I believe someone from us is already working on this and we will hopefully be able to address this soon.

**Comment by Huawei-HiQ** [comment]
Yes, this function has been implemented, and it will take a while to push the code.

**Comment by Neuromancer43** [comment]
This issue has been solved in HiQ 2.0.  😃
